1.0/docs/devguide/properties.md

```diff
@@ -422,70 +422,44 @@ Example:
       </script>
     </dom-module>
 
-### Deep sub-property observation {#deep-observation}
-
-To call an observer when any (deep) sub-property of an
-object changes, specify a path with a wildcard (`*`).
-
-When you specify a path with a wildcard, the argument passed to your
-observer is a change record object with the following properties:
+### Observe array mutations {#array-observation}
 
-*   `path`. Path to the property that changed. 
-*   `value`. New value of the path that changed.
-*   `base`. The object matching the non-wildcard portion of the path. 
+Use a splice observer to call an array mutation observer whenever an array 
```

Comment:
"Use a splice observer to call an array mutation observer ..." is confusing--they're the same thing. I prefer the term "array mutation observer".

Suggest:
"Use an array mutation observer to call an observer function whenever ..."